pulser/sequence.py

```diff
@@ -1051,7 +1061,9 @@ def _target(
                     delta = self._channels[channel].validate_duration(
                         16 if delta < 16 else delta
                     )
-            tf = ti + delta
+            tf = ti + max(
+                delta, cast(int, self._channels[channel].fixed_retarget_t)
+            )
```

Comment:
Shouldn't this also take into account `retarget_time`?
Do you confirm that retargetting and phase jumping can be performed in parallel?

Reply:
`retarget_time` is already considered in the `delta` value. 

Yes, retargeting and phase jumping can be performed in parallel because they take place on different hardware components.

Reply:
It's actually the delta between the two retarget ends that matters, note that you always keep `ti` the same and shift `tf` to satisfy your criteria. This also means that if `fixed_retarget_t > retarget_time`, then `retarget_time` is irrelevant because it will always be assured by definition. Does it make sense now?

Reply:
Yes, makes perfect sense. It would probably be worth it to precise that the interval is between both ends on the variable's comment (‶interval between instructions″ might be confusing).

Reply:
Ok, I changed it too.